CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -36,6 +36,10 @@ export default class MobileNavDraw extends React.Component<IMobileNavDrawProps,
             <li><a href="/find">Find Project</a></li>
             <li><a href="/start">Start Project</a></li>
             <li><a href="/about">About</a></li>
+            {this.props.loginText == "Logout" ?
```

Comment:
This whole thing could be solved a bit more neatly. There are two if statements (checking for "Logout"), and they're beneath each other.

First, make these one proper if statement, not these weird ternary statements: https://reactjs.org/docs/conditional-rendering.html

It's a pre-existing problem with this code, but loginText should be a boolean, since it's treated that way. Maybe call it "isLoggedIn"?